programmers/2일차/수박수박수박수.js

```diff
@@ -0,0 +1,23 @@
+function solution (n) {
+    const word  = "수박";
+    const split_word = word.split("");
+    const answer =[];
```

Comment:
```suggestion
    const answers =[];
```
배열이나 객체는 여러가지 값을 담고 있기에  변수명을 복수로 선언을 해주는게 정석입니다^^